onnx/__init__.py

```diff
@@ -49,38 +53,39 @@
 from .version import version as __version__  # noqa
 
 # Import common subpackages so they're available when you 'import onnx'
-import onnx.checker  # noqa
-import onnx.defs  # noqa
-import onnx.helper  # noqa
-import onnx.utils  # noqa
-import onnx.compose  # noqa
-
-import google.protobuf.message
-
-from typing import Union, IO, Optional, cast, TypeVar, Any
+from . import (
+    checker,
+    compose,
+    defs,
+    gen_proto,
+    helper,
+    hub,
```

Comment:
@jcwchen i didn't look to make sure the sub modules have imports that may take too long. are there modules with a long initialization time we should avoid importing here?

Reply:
Sounds good! Feel free to merge